config.tf.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Here's a file will be used for `terraform apply` @faultables

Reply:
`{}` bakalan terisi berdasarkan `modules = []` pada flake.nix:31:33 @faultables

Reply:
nah eventually harus jalanin `nix build` ini somewhere kan (local/CI/atlantis server)

Reply:
@faultables  development mode generated config.tf.json trus push ke repository ini. 

atlantis referensinya kerepo ini. so, `terraform apply` jadi possible diatlantis.